hws/mlx_hw_steering_parser.py

```diff
@@ -195,7 +207,9 @@ def parse_args():
                         help="Trigger DPDK app <PID>.")
     parser.add_argument("-port", dest="dpdk_port", type=int, default=0,
                         help="Trigger DPDK app <PORT> newer dpdk supports -1 for all ports (must provide PID with -pid).")
-    parser.add_argument('-h', '--help', action='help', default=argparse.SUPPRESS,
+    parser.add_argument("-extra_hw_res", type=str, default="", dest="extra_hw_res", metavar="[pat, arg]",
+                        help = "Provide extra HW resources to be dumped. for example: -extra_hw_res pat,arg")
```

Comment:
"Provide extra HW resources to be dumped"
Perhaps "require"? "request"? We're not providing - we're just asking to dump more resources